apps/src/gamelab/AnimationPicker/actions.js

```diff
@@ -12,9 +12,11 @@ var AnimationPickerAction = module.exports.AnimationPickerAction = {
  * Show the animation picker
  * @returns {{type: AnimationPickerAction}}
  */
-module.exports.showAnimationPicker = function () {
+module.exports.showAnimationPicker = function (onComplete, onCancel) {
   return {
-    type: AnimationPickerAction.SHOW_ANIMATION_PICKER
+    type: AnimationPickerAction.SHOW_ANIMATION_PICKER,
+    onComplete: onComplete,
+    onCancel: onCancel
```

Comment:
That argument makes sense.  I suppose those functions don't belong in the store at all.

The thing I haven't wrapped my head around yet, is how to let my AnimationTab code treat the dialog like an async operation when it's being 'stateless'-rendered by React.  Any ideas?